lib/sqlalchemy/ext/compiler.py

```diff
@@ -85,8 +85,8 @@ def __init__(self, table, select):
     @compiles(InsertFromSelect)
     def visit_insert_from_select(element, compiler, **kw):
         return "INSERT INTO %s (%s)" % (
-            compiler.process(element.table, asfrom=True),
-            compiler.process(element.select)
+            compiler.process(element.table, asfrom=True, **kw),
```

Comment:
there's a thing that can happen sometimes with this which is that the kw would already have "asfrom" inside of it, and this would raise an error.  however insertfromselect in this example would not be embedded inside a FROM clause itself so we can keep this as is.  we would not want asfrom=True for the element.select visit either.